src/backend/marsha/settings.py

```diff
@@ -107,6 +107,7 @@ class Base(Configuration):
     # Application definition
 
     INSTALLED_APPS = [
+        "scout_apm.django",
```

Comment:
Is it required to put it first in the list? If so, a comment would have be nice.